### DISCUSSION TITLE
Bug/Enh: Searching

### DISCUSSION THREAD

**Discussion by RealMrCactus** [open-webui/open-webui#2963] [General]
# Bug Report

## Description

**Bug Summary:**
When using multiple models at once in the same chat they try to spawn individual search instances instead of using 1 for how ever many there are

**Steps to Reproduce:**
Enable web search
Select Two LLMs
Prompt them
watch as they make individual searches

**Expected Behavior:**
They use the same search

**Actual Behavior:**
They try to use their own searches

## Environment

- **Open WebUI Version:** 0.2.4

- **Operating System:** Ubuntu 22.04

## Reproduction Details

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [ ] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the Docker container logs.

## Logs and Screenshots

**Screenshots (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

![image](https://github.com/open-webui/open-webui/assets/36554881/354aebe7-361c-45a5-a20c-9c6aec106234)
> [!NOTE]  
> The other one shows no results found as the brave search free plan for their api only allows 1 search a second iirc

## Installation Method

Docker

> [!NOTE]  
> Was unsure if this was a bug or intended so i threw enh in the title as well

**Comment by tjbck** [comment]
Can't seem to reproduce with the latest :/

![image](https://github.com/open-webui/open-webui/assets/25473318/64186f63-6cca-4795-bbd2-0eebb0ff58ca)

**Reply by RealMrCactus** [reply]
![image](https://github.com/open-webui/open-webui/assets/36554881/ffd3f0fe-3328-4dd3-86d4-6ca9de5f28be)
 still does it for me on latest (just updated)
